src/global/graphqls/schema.ts

```diff
@@ -0,0 +1,58 @@
+import { WithTypeName } from 'typings';
+import * as Yup from 'yup';
+
+import { ImageRef } from 'components/CropperWidget/CropperWidget';
+
+export const cardSchema = Yup.object({
+  name: Yup.string().required(),
+  description: Yup.string().required(),
+  avatar: Yup.string().required(),
+  background: Yup.string().required(),
+  avatarUpload: Yup.mixed<ImageRef>()
+    .nullable(true)
+    .notRequired(),
+  backgroundUpload: Yup.mixed<ImageRef>()
+    .nullable(true)
+    .notRequired(),
+});
+
+export const pageSchema = Yup.object({
+  content: Yup.mixed()
+    .required()
+    .nullable(true),
+  avatar: Yup.string().required(),
```

Comment:
avatar and background are not required fields according to the requirements, right @pumanitro ?